packages/shorebird_cli/integration_test/shorebird_cli_integration_test.dart

```diff
@@ -1,19 +1,104 @@
 import 'dart:io';
 
+import 'package:mason_logger/mason_logger.dart';
+import 'package:path/path.dart' as p;
 import 'package:test/test.dart';
 
 void main() {
-  test('--version', () async {
-    final result = await Process.run(
+  final logger = Logger();
+
+  ProcessResult runShorebirdCommand(
+    String command, {
+    required String workingDirectory,
+  }) {
+    logger.info('running shorebird $command in $workingDirectory');
+    return Process.runSync(
       'shorebird',
-      ['--version'],
+      command.split(' '),
       runInShell: true,
+      workingDirectory: workingDirectory,
     );
+  }
+
+  test('--version', () {
+    final result = runShorebirdCommand('--version', workingDirectory: '.');
     expect(result.stderr, isEmpty);
     expect(
       result.stdout,
       stringContainsInOrder(['Shorebird Engine', 'revision']),
     );
     expect(result.exitCode, 0);
```

Comment:
```suggestion
    expect(result.exitCode, equals(0));
```

nit but imo it's a bit more readable to use the `equals` matcher similar to how we use `isTrue`/`isFalse`.

Reply:
Done.